analyzers/its/expected/Ember-MM/generic.EmberCore.NMT-{84B2143A-D04F-4262-923D-21AEDF86E2B7}-S2259.json

```diff
@@ -158,19 +158,6 @@
 },
 {
 "id":  "S2259",
-"message":  "'c.values.FirstOrDefault(Function(y) y.label = saved)' is Nothing on at least one execution path.",
-"location":  {
-"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
-"region":  {
-"startLine":  960,
```

Comment:
Lost TP. Not sure why, the method is not so big

Reply:
Not clear to me why this is a TP :/

Reply:
`FirstOrDefault` returns something or `Nothing`. On the `Nothing` case, it's TP